build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+set -e
+
+cd "$(dirname "$0")"
+
+go build -o ./bin/openshift-install ./cmd/openshift-install
```

Comment:
If we want static binaries (and [I think we do, but...][1]), you'll want `CGO_ENABLED=0` here.

[1]: https://github.com/openshift/installer/pull/326#discussion_r220371789